scalableminds/webknossos
# Improve task list

### Steps to test:
- Go to task list and search to get some results.
- Make sure tasks are sorted by project date, but from latest to earliest.
- Test "expand all" button
- Check that expanded rows look nice 

### Issues:
- fixes #7586 

------
(Please delete unneeded items, merge only when none are left open)
